.github/workflows/citgm-package.yml

```diff
@@ -6,12 +6,79 @@ on:
       package:
         description: 'Package to test'
         required: true
-        type: string
+        type: choice
+        options:
```

Comment:
what if someone wants to test the package not on the list?

Reply:
oh, you are correct.